syslog-ng-helm-chart/values.yaml

```diff
@@ -0,0 +1,107 @@
+# Default values for syslog-ng.
+
+replicaCount: 1
+
+image:
+  repository: balabit/syslog-ng
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "3.27.1"
+
+# By default, the network() driver binds to 0.0.0.0, meaning that it listens 
+# on every available IPV4 interface on the TCP/601 port.
+
+config: |
+  @version: 3.27
+  @include "scl.conf"
+  options {
+    # enable or disable directory creation for destination files
+    create_dirs(yes);
+
+    # keep hostnames from source host
+    keep_hostname(yes);
+
+    # use ISO8601 timestamps
+    ts_format(iso);
+  };
+  log {
+  	source {
+  		network();
+  	};
+  	destination { file("/var/log/${YEAR}-${MONTH}-syslog"); };
+  };
+
+storage:
+  enable: True
+  storageClass: netapp
```

Comment:
What is "netapp" here? Is it a setting that makes sense for a broad set of users or it's local to you?

Reply:
Ups! that was a local thing to me. i left it blank so k8s will pick up the default sotrageclass.